src/main/java/nextstep/fp/Lambda.java

```diff
@@ -27,30 +27,30 @@ public void run() {
     }
 
     public static int sumAll(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            total += number;
-        }
-        return total;
+        return numbers.stream()
+                .mapToInt(number -> number)
+                .sum();
     }
 
     public static int sumAllEven(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            if (number % 2 == 0) {
-                total += number;
-            }
-        }
-        return total;
+        return numbers.stream()
+                .mapToInt(number -> number)
+                .filter(number -> number % 2 == 0)
+                .sum();
     }
 
     public static int sumAllOverThree(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            if (number > 3) {
-                total += number;
-            }
-        }
-        return total;
+        return numbers.stream()
+                .mapToInt(number -> number)
+                .filter(number -> number > 3)
+                .sum();
     }
+
+    public static int sumAll(List<Integer> numbers, Conditional c) {
```

Comment:
변수명은 축약하지 않고 선언하면 어떨까요?
앞서 드린 코멘트에서 말씀드린 sumAll 메서드가 이 메서드입니다.
접근 제한자를 private으로 변경하고 활용해 보시면 좋을 것 같아요 
```suggestion
    private static int sumAll(List<Integer> numbers, Conditional condition) {
```